README.md

```diff
@@ -145,6 +145,7 @@ files, URLs, or add other options.
 * `golang` stack
 * `swift` compiler
 * `clang` and Objective-C Foundation (must be tested separately on macOS)
+# `rust` tools
```

Comment:
This should be a bullet-point not a header, right?

Reply:
Good catch.

Reply:
Certainly, thanks.